app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,10 @@
 class WelcomeController < ApplicationController
   def index
+    if current_user
```

Comment:
That's what the checkpoint asked for---the default for a signed in user is to see the user#show page, rather than the landing page.